src/components/AddItems.js

```diff
@@ -89,70 +92,85 @@ const AddItemsToList = () => {
   }
 
   return (
-    <div>
-      <div>
+    <div className="flex justify-center ">
+      <div className=" text-black md:mt-40 md:w-1/3 mt-20">
         <form onSubmit={submitShoppingListItemHandler}>
-          <h2>Add Item to List</h2>
           {shoppingListItemNameExists ? (
             <p>
               {`You have ${normalizeString(
                 shoppingListItemName,
               )} in your shopping list already`}
             </p>
           ) : null}
-          <div>
-            <label>
-              Name
+          <div className="">
+            <div className="flex justify-center">
+              {/* <label className="text-black">Name of Item</label>  */}
+
               <input
                 type="text"
+                placeholder="Add Item..."
                 value={shoppingListItemName}
                 onChange={shoppingListItemNameHandler}
+                className="border  text-grey-darkest md:w-2/3 px-2 py-2 md:px-4 md:py-3 mb-8 rounded"
               />
-            </label>
+            </div>
           </div>
-          <fieldset>
-            How soon are you likely to buy it again?
-            <div>
+          <div className="flex justify-center md:mb-8 mb-4">
```

Comment:
Again, why using flex? No need for flex and justify-center just for the p tag.

Reply:
Ah great, thanks for the suggestion Julia, I've amended and it works perfectly!